.github/workflows/ci.yml

```diff
@@ -180,10 +180,10 @@ jobs:
       - name: Setup rust toolchain
         run: rustup show
       - uses: Swatinem/rust-cache@v2.0.0
-
       - name: Install sqlx-cli
+        # Lock sqlx-cli version to the same version as sqlx in the workspace
         run: |
-          cargo install sqlx-cli
+          cargo install sqlx-cli@0.6.1
```

Comment:
Maybe lock it to the latest (0.6.2) and push the changes as well?

Reply:
sure, but this makes sense only when sqlx gets upgraded. upgrading it now.